apps/condo/domains/organization/schema/OrganizationEmployeeRequest.js

```diff
@@ -247,6 +269,18 @@ const OrganizationEmployeeRequest = new GQLListSchema('OrganizationEmployeeReque
                 }
             }
 
+            const userId = get(resolvedData, 'user')
```

Comment:
We can also double check that authorized user is the one that is in resolvedData, so that I cannot possbily know other people's phone numbers

Reply:
I think we can pass it as is, because user cannot create/update `OrganizationEmployeeRequest` directly (only soft delete), he can do it only with `SendOrganizationEmployeeRequestService`, where an `authedItem` is passed in the `user` field.
And in many tests in `OrganizationEmployeeRequest` admin client creates requests for other users..

Reply:
agreed, but I'd have added a second check... If im not mistaken it is practicly free (context.auth.user)